### ISSUE TITLE
Open the Camera on start

### ISSUE THREAD

**Issue by ahmedalam** [uzysjung/UzysAssetsPickerController#52]
Hi, Is it possible to open the camera when initialised rather then the assets library?
thanks.

**Comment by gankac** [comment]
Any Luck on this @ahmedalam .

**Comment by ahmedalam** [comment]
Nope.. I ended up making my own implementation of camera and gallery picker :)
